### DISCUSSION TITLE
how can I get record id after txDao.SaveRecord(record) runs success.

### DISCUSSION THREAD

**Discussion by demirdev** [pocketbase/pocketbase#1668] [Q&A] [answered]
In my app I have clients and their balances. 
I want to create clients and balances in db transaction.
After I saved client, I cannot get client by txDao.FindFirstRecordByData.

After obtain client id, I will create record for balances collection with relation with client id.

Thanks a lot.
Best regards.
🙂👍

**Comment by ganigeorgiev** [comment, marked as answer]
`Dao.SaveRecord()` works with a `models.Record` pointer, so you should be able to access the record id by just reading it from the model, aka. `record.Id`.

If that's not what you are asking, could you elaborate more what you are trying to do and provide an example/code sample?

**Reply by demirdev** [reply]
Thank you very much. For fast and true answer. 
Best regards.

👏👏👏
